content/en/docs/Deployment/kubernetes-deploy.md

```diff
@@ -145,6 +145,24 @@ Kubernetes namespace.
 To activate this alternative, use the option
 `--set features.eirini.enabled=true` when deploying kubecf from its chart.
 
+#### Diego Cell Affinities & Tainted Nodes
+
+Note that the `diego-cell` pods used by the Diego standard scheduler
+are
+
+  - privileged,
+  - use large local emptyDir volumes (i.e. require node disk storage),
+  - and set kernel parameters on the node.
```

Comment:
This is (partially) why they need to be privileged. Noting it here is probably too much detail, though.

Reply:
Maybe rephrase, move the info about privileged to the line about kernel params?

Reply:
Nah, this is probably fine.